cookbook/index.rst

```diff
@@ -16,3 +18,5 @@ Awesome recipies for lunch
     integrating-custom-filters
     extend-entities
     using-elasticsearch
+
+It is possible to work through the recepies, altought most of the people will pick the ones, which are most similar to their own tasks. We're open to `suggestions <http://sulu.io/en/contact>`_.
```

Comment:
80 chars, and if the syntax with the `_` at the end is used, the link has to be defined at the end, shouldn't it?